app/views/submission_filters/_list.html.erb

```diff
@@ -0,0 +1,57 @@
+<table class="submissions-table">
+  <thead class="submissions-table-header">
+    <tr>
+      <% if columns.include?(:full_name) %>
```

Comment:
@hanka Would you do it like this or use a Hash and have `columns[:full_name]`? The hash would require not only saying which columns we want (column: true), but also which we don't - would be a little longer. And the columns array will always be short so performance is not an argument...